app/design/adminhtml/Magento/backend/web/css/source/forms/_fields.less

```diff
@@ -121,6 +121,7 @@
 
         > .admin__field-control {
             #mix-grid .column(@field-control-grid__column, @field-grid__columns);
+            padding-top: 7px;
```

Comment:
If the change is actual only for bundle product form, it would be better to use more strict selector